### PR TITLE
Conversion validation

### PR DESCRIPTION
**Pull request by yozik04**
I want to verify if json files are converted correctly from CSV files inside pull requests.

### CODE REVIEW

**Review by yozik04 — COMMENTED** [review]
With conversion mappings from Series to a dictionary I need now to fix order of mappings in the json.

File: nibe/data/vvm310_vvm500.json [hunk 1]
```diff
@@ -2604,7 +2604,11 @@
     "info": "Indicates active relays on the PCA Base card. Please refer to the wiring diagram for relay description. Binary encoded. 1=on, 0=off. Bit0=K4,Bit1=K3,Bit2=K2,Bit3=K1",
     "size": "u8",
     "factor": 1,
-    "name": "eb100-ep14-pca-base-relays-43514"
+    "name": "eb100-ep14-pca-base-relays-43514",
+    "mappings": {
+      "0": "off",
+      "1": "on"
+    }
```
Comment (elupus) [inline nibe/data/vvm310_vvm500.json:2611]: I think this might be wrong. I think these are bitfields, but maybe the csv has invalid max values?

Comment (yozik04) [reply nibe/data/vvm310_vvm500.json]: Yes. It is wrong. Dropped all mappings that are binary encoded.

**Comment by yozik04** [comment]
@elupus, you want to review it? I think it is fine as no JSON files were changed.

**Review by elupus — COMMENTED** [review]
Not reviewed in detail, but cursory it looks good

**Pull request merged**